net-p2p/resilio-sync/resilio-sync-2.6.1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BASE_URI="http://linux-packages.resilio.com/${PN}/deb/pool/non-free/r/${PN}/${PN}_${PV}-1_@arch@.deb"
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles unpacker user
+
+NAME="rslsync"
+QA_PREBUILT="usr/bin/${NAME}"
+
+DESCRIPTION="Resilient, fast and scalable file synchronization tool"
+HOMEPAGE="https://resilio.com/"
+SRC_URI="amd64? ( ${BASE_URI/@arch@/amd64} )
+	x86? ( ${BASE_URI/@arch@/i386} )"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="pax_kernel"
+RESTRICT="mirror"
+
+S="${WORKDIR}"
+
+DOC_CONTENTS="You may need to review /etc/${PN}/config.json\\n
+Default metadata path is /var/lib/${PN}/.sync\\n
+Default web-gui URL is http://localhost:8888/\\n\\n"
+
+pkg_setup() {
+	enewgroup "${NAME}"
+	enewuser "${NAME}" -1 -1 /var/lib/"${PN}" "${NAME}"
+}
+
+src_unpack() {
+	unpacker_src_unpack
+
+	unpack usr/share/man/man1/"${PN}".1.gz
+}
+
+src_install() {
+	dobin usr/bin/"${NAME}"
+	use pax_kernel && pax-mark m "${ED%/}"/usr/bin/"${NAME}"
+
+	doman "${PN}".1
+
+	dodir /var/log/"${PN}"
+	keepdir /etc/"${PN}" /var/lib/"${PN}"/ /var/lib/"${PN}"/.sync /var/log/"${PN}"
+	fperms 0700 /etc/"${PN}" /var/lib/"${PN}" /var/lib/"${PN}"/.sync /var/log/"${PN}"
```

Comment:
Instead of changing permissions and ownership, use `diropts` to set them before creating those directories. And then move it to the bottom, so that  they don't affect other stuff you do here.